README.md

```diff
@@ -76,7 +76,7 @@ Documentation
 Docker images
 -------------
 
-Docker images are available on [Quay.io](https://quay.io/organization/topolvm)
+Docker images are available on [ghcr.io](https://github.com/orgs/topolvm/packages)
```

Comment:
Please add a note that images on quay.io are now deprecated and will be unavailable in the future.